indexclient/client.py

```diff
@@ -88,6 +89,32 @@ def get(self, did):
 
         return Document(self, did, json=response.json())
 
+    def bulk_request(self, dids):
+        """
+        bulk_get makes one http request to the indexd service and retrieves
+        a list of Documents based on the dids provided.
+
+        Args:
+            dids (list): list of dids for potential documents
+
+        Returns:
+            list: Document objects representing  index records
+        """
+
+        headers = {'content-type': 'application/json'}
+        try:
+            response = self._post("bulk/documents", json=dids, headers=headers)
+        except requests.HTTPError as exception:
```

Comment:
Can you confirm that if one of the did's doesn't exist it doesn't get eaten by this?